emarsys/api.py

```diff
@@ -95,7 +100,7 @@ def call(self, uri, method, params=None):
                 "replyText" in result and "data" in result):
             message = u"Unexpected response from Emarsys"
             if not response.ok:
-                message = u"{message} (HTTP {status_code})".format(
+                message = u"{message} (HTTP {code})".format(
```

Comment:
It wasn't before. Interestingly github was smart enough to understand the commit split and sorted the comment back to the proper line and the right commit.